tendrl/gluster_integration/sds_sync/utilization.py

```diff
@@ -40,27 +40,53 @@ def sync_utilization_details(volumes):
     out = ''
     try:
         out, err = cmd.communicate()
+        
         util_det = json.loads(out)
-        for k, v in util_det.iteritems():
-            volume = voldict[k]
-            volume.usable_capacity = int(v['total'])
-            volume.used_capacity = int(v['used'])
-            volume.pcnt_used = str(v['pcnt_used'])
-            volume.total_inode_capacity = int(v['total_inode'])
-            volume.used_inode_capacity = int(v['used_inode'])
-            volume.pcnt_inode_used = str(v['pcnt_inode_used'])
-            volume.save()
-            cluster_used_capacity += volume.used_capacity
-            cluster_usable_capacity += volume.usable_capacity
-    except(KeyError, TypeError, ValueError, AttributeError):
+        err_dict = json.loads(err)
+
+        if err_dict != '':
```

Comment:
I feel err_dict is a dictionary and if so why we check `if err_dict != ''` ?
Better we should have check like `if bool(err_dict):`